File: Recommender/src/main/java/cn/edu/ahtcm/caseclass/recs/UserRecs.scala
package cn.edu.ahtcm.caseclass.recs

/**
 * 定义用户的推荐列表
 *
 * @param userId        用户 ID
 * @param recsProductId 推荐商品 ID
 * @param score         评分
 */
case class UserRecs(userId: Int, recsProductId: Int, score: Double)

File: Mall/src/main/java/cn/edu/ahtcm/mall/bean/UserRecs.java
package cn.edu.ahtcm.mall.bean;

public class UserRecs {

    private int userId;
    private int recsProductId;
    private Double score;

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getRecsProductId() {
        return recsProductId;
    }

    public void setRecsProductId(int recsProductId) {
        this.recsProductId = recsProductId;
    }

    public Double getScore() {
        return score;
    }

    public void setScore(Double score) {
        this.score = score;
    }
}
